models/model-cards/gnn-fsi-modelcard.md

```diff
@@ -0,0 +1,183 @@
+
+## Model Architecture:
+It uses a bipartite heterogeneous graph representation as input for `GraphSAGE` for feature learning and `XGBoost` as a classifier. Since the input graph is heterogeneous, a heterogeneous implementation of `GraphSAGE` (HinSAGE) is used for feature embedding.<br>
+**Architecture Type:** Graph Neural Network and Binary classification <br>
+**Network Architecture:** GraphSAGE and XGBoost <br>
+
+## Input
+Transaction data with nodes including transaction, client, and merchant.<br>
+**Input Parameters:** None <br>
+## Output
+An anomalous score of transactions indicates a probability score of being a fraud.<br>
+**Output Parameters:** None <br>
+## Software Integration:
+**Runtime(s):** Morpheus  <br>
+
+**Supported Hardware Platform(s):** <br>
+* Ampere/Turing <br>
+
+**Supported Operating System(s):** <br>
+* Linux <br>
+  
+## Model Version(s): 
+* 1.0 <br>
+
+### How To Use This Model
+This model is an example of a fraud detection pipeline using a graph neural network and gradient boosting trees. This can be further retrained or fine-tuned to be used for similar types of transaction networks with similar graph structures.
+
+# Training & Evaluation: 
+
+## Training Dataset:
+
+**Link:** [fraud-detection-training-data.csv](models/dataset/fraud-detection-training-data.csv)  <br>
+**Properties (Quantity, Dataset Descriptions, Sensor(s)):** A training data consists of raw 753 labeled credit card transaction data with data augmentation in a total of 12053 labeled transaction data. <br>
+**Dataset License:**  (N/A) <br>
+
+## Evaluation Dataset:
+**Link:**  [fraud-detection-validation-data.csv](models/dataset/fraud-detection-validation-data.csv)  <br>
+**Properties (Quantity, Dataset Descriptions, Sensor(s)):** Data consists of raw 265 labeled credit card transaction synthetically created<br>
+**Dataset License:** (N/A)<br>
+
+## Inference:
+**Engine:** Triton
+**Test Hardware:** <br>
+* Other (Not Listed)  <br>
+
+## Model Card ++ Bias Subcard
+
+### What is the gender balance of the model validation data?  
+* Not Applicable
+
+### What is the racial/ethnicity balance of the model validation data?
+* Not Applicable
+
+### What is the age balance of the model validation data?
+* Not Applicable
+
+### What is the language balance of the model validation data?
+* Not Applicable
+
+### What is the geographic origin language balance of the model validation data?
+* Not Applicable
+
+### What is the educational background balance of the model validation data?
+* Not Applicable
+
+### What is the accent balance of the model validation data?
+* Not Applicable
+### Describe measures taken to mitigate against unwanted bias.
+* Not Applicable
+## Model Card ++ Explainability Subcard
+
+### Name example applications and use cases for this model. 
+* This model is intended to be used for fraud detection use cases.
```

Comment:
The model card should describe the pretrained model files (the pretrained weights) saved in the Morpheus repo, not so much as the GNN model itself as a pipeline/concept.
Is the pretrained model readily useful for other users (e.g. like the Word2Vec model)? Or is the model file used primarily as an example and/or for testing purposes?
If it's used mainly for testing, the answers I wrote for DFP in this section might be largely applicable here as well: https://github.com/nv-morpheus/Morpheus/blob/d1a13dc6edaaaa9505d185d5554068337fe2ea00/models/model-cards/dfp-model-card.md